File-Formats/DocIO/Convert-Word-Document-to-PDF-in-UWP.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Convert Word document to PDF in UWP | Syncfusion
+description: Convert Word document to PDF without Microsoft Word or interop dependencies in UWP application using .NET Core Word (DocIO) library
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in UWP
+
+Syncfusion Essential DocIO is a [UWP Word library](https://www.syncfusion.com/document-processing/word-framework/uwp/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to PDF in UWP**.
```

Comment:
Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) ...